packages/plugin-commands-licenses/src/licenses.ts

```diff
@@ -0,0 +1,182 @@
+import {
+  docsUrl,
+  readDepNameCompletions,
+  readProjectManifestOnly,
+} from '@pnpm/cli-utils'
+import { CompletionFunc } from '@pnpm/command'
+import { WANTED_LOCKFILE } from '@pnpm/constants'
+import { readWantedLockfile } from '@pnpm/lockfile-file'
+import {
+  FILTERING,
+  OPTIONS,
+  UNIVERSAL_OPTIONS,
+} from '@pnpm/common-cli-options-help'
+import { Config, types as allTypes } from '@pnpm/config'
+import { PnpmError } from '@pnpm/error'
+import { licences } from '@pnpm/licenses'
+import pick from 'ramda/src/pick'
+import renderHelp from 'render-help'
+import { renderLicences } from './outputRenderer'
+
+export function rcOptionsTypes () {
+  return {
+    ...pick(
+      [
+        'depth',
+        'dev',
+        'global-dir',
+        'global',
+        'json',
+        'long',
+        'optional',
+        'production',
+      ],
+      allTypes
+    ),
+    compatible: Boolean,
+    table: Boolean,
+  }
+}
+
+export const cliOptionsTypes = () => ({
+  ...rcOptionsTypes(),
+  recursive: Boolean,
+})
+
+export const shorthands = {
+  D: '--dev',
+  P: '--production',
+}
+
+export const commandNames = ['licenses']
+
+export function help () {
+  return renderHelp({
+    description: `Check for licenses packages. The check can be limited to a subset of the installed packages by providing arguments (patterns are supported).
+
+Examples:
+pnpm licenses
+pnpm licenses --long
+pnpm licenses gulp-* @babel/core`,
+    descriptionLists: [
+      {
+        title: 'Options',
+
+        list: [
+          {
+            description:
+              'By default, details about the outdated packages (such as a link to the repo) are not displayed. \
```

Comment:
what is an outdated package? Have you copied the options from a different command without changing the text?

Reply:
Yeah, I used one of the existing commands as a base, I have cleaned up the command line descriptions/options